pkg/logging-reconciler/logging_reconciler.go

```diff
@@ -62,10 +53,9 @@ func (l *LoggingReconciler) reconcileCreate(ctx context.Context, lc loggedcluste
 
 	// Call all reconcilers ReconcileCreate methods.
 	for _, reconciler := range l.Reconcilers {
-		// TODO(theo): add handling for returned ctrl.Result value.
-		_, err := reconciler.ReconcileCreate(ctx, lc)
-		if err != nil {
-			return ctrl.Result{}, errors.WithStack(err)
+		result, err := reconciler.ReconcileCreate(ctx, lc)
+		if err != nil || !result.IsZero() {
```

Comment:
If there is one error or the returned result is not empty (either requeue or requeueafter then we return to close the reconciliation loop)